### DISCUSSION TITLE
doesnt record

### DISCUSSION THREAD

**Discussion by i1o1** [w-okada/voice-changer#726] [Q&A]
i followed duckus' tutorial on this and it gets set up and runs fine, but it doesnt playback any audio, or record any audio at all and i have no idea why. help?

**Comment by FernThatCodes** [comment]
I'm assuming you are talking about the audio not playing back on your device? If that's the case, check if the output setting under audio is set to a speaker or headphones. If you can provide a screenshot I might be able to spot the problem. If none of that works I suggest reinstalling and trying again with this tutorial (https://youtu.be/_JXbvSTGPoo)
